jquery.counterup.js

```diff
@@ -14,7 +14,8 @@
     // Defaults
     var settings = $.extend({
         'time': 400,
-        'delay': 10
+        'delay': 10,
+        'beginAt': 1
```

Comment:
Good point. Updated in https://github.com/bfintal/Counter-Up/pull/45/commits/e61066cc88d385dcf09a1ae897cc4bc8322bb9cf